src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {

```

Comment:
I think Figure its something that has state, maybe its better to make figure as abstract class, not interface? And lets create 2 interfaces for `getArea()` and `draw()` methods (some `AreaCalculator` and `Drawer` interfaces, or smth like this)